src/__tests__/BranchName.js

```diff
@@ -4,26 +4,14 @@ import {render} from '../utils/testing'
 import {COMMON} from '../system-props'
 
 describe('BranchName', () => {
-  it('is a system component', () => {
-    expect(BranchName.systemComponent).toEqual(true)
-  })
-
   it('renders an <a> by default', () => {
     expect(render(<BranchName />).type).toEqual('a')
   })
 
-  it('renders class="branch-name"', () => {
```

Comment:
Converted this component to be all css in js so these tests are no longer needed